platform_channels/test/src/method_channel_demo_test.dart

```diff
@@ -3,18 +3,47 @@
 // found in the LICENSE file.
 
 import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
 import 'package:flutter_test/flutter_test.dart';
 import 'package:platform_channels/src/method_channel_demo.dart';
 
 void main() {
+  TestWidgetsFlutterBinding.ensureInitialized();
```

Comment:
This shouldn't be necessary. Try taking it out and see if everything still works. :)